src/site/content/en/blog/stories-component/index.md

```diff
@@ -25,36 +36,54 @@ Let's explore and get inspired by this minimal way to layout an interactive Stor
   </video>
 </figure>
 
-### Introduction
-Stories are commonly found as a mobile only, tap centric pattern for navigating multiple subscriptions. For example: on Instagram, users open a friend's story and go through the pictures in it. They generally do this many friends at a time. By tapping on the right side of the device, users move forward through "their stories," which are their friends and their friends' stories.
+If you would prefer a hands-on demonstration of building this Stories component yourself,
+check out the [Stories component codelab](TODO).
```

Comment:
Since we're doing a pretty cool thing here and providing a lot of different learning paths we should make them aware of that early on